ospfd/ospf_vty.c

```diff
@@ -8831,6 +8833,7 @@ DEFUN (no_ip_ospf_area,
 		return CMD_NOT_MY_INSTANCE;
 
 	argv_find(argv, argc, "area", &idx);
+	VTY_GET_OSPF_AREA_ID(area_id, format, argv[idx + 1]->arg);
```

Comment:
Area ID is an optional parameter, you can't use `VTY_GET_OSPF_AREA_ID` here.
I suppose you can get configured area ID from the `params` struct later in the code.